### ISSUE TITLE
event onAnyMessage alway get old message received when first time run

### ISSUE THREAD

**Issue by ductai230894** [reo7sp/tgbot-cpp#223]
Hello sir,
I have problem with onAnyMessage, when I run, It always get old message first, any way to prevent this

**Comment by llnulldisk** [comment]
Hello,
what exactly do you mean?
A user writes several messages while the bot is offline. Then the bot is started and the messages are received from old to new with onAnyMessage(). You want to receive the latest messages first?

**Comment by ductai230894** [comment]
No, this message already read by bot, but when run start again, bot continue read this message, this is old message, I test with listen message from channel, maybe I'm wrong, but bot always read old message which Bot already read it

**Comment by llnulldisk** [comment]
This shouldn't happen. Do you have any example code to reproduce this problem?

**Comment by ductai230894** [comment]
this is example code, bot is admin in channel, bot already read message in channel, but when I run application again, bot continue read this message, I don't know why..
wait me 30 minute, I will capture video

**Comment by yourAnonDH** [comment]
above of `while (1) { longpoll.start() }` write this `bot.getApi().deleteWebhook();`

**Comment by reo7sp** [comment]
Telegram keeps unconfirmed updates while the bot is offline, and a new process does not retain the previous offset. If discarding pending updates is intentional, call `bot.getApi().deleteWebhook(true)` before polling.
